lib/merit/controller_extensions.rb

```diff
@@ -47,7 +47,7 @@ def had_errors?
     end
 
     def target_object
-      target_obj = instance_variable_get(:"@#{controller_name.singularize}")
+      target_obj = instance_variable_defined? ? instance_variable_get(:"@#{controller_name.singularize}") : nil
```

Comment:
Thar version can probably improved, or we might be able to extract another method off of it.